execution-environment.yml

```diff
@@ -56,6 +56,8 @@ dependencies:
 additional_build_steps:
   append_base:
     - RUN $PYCMD -m pip install -U pip
+  prepend_builder:
+    - RUN $PKGMGR install -y cmake gcc gcc-c++ make openssl-devel # needed for building uamqp for azure.azcollection on arm64
```

Comment:
Is there a reason you put this here instead of listing these under here? 

https://github.com/ansible/awx-ee/blob/71158e4db0dcfe06e5a49e123269ce76a2a5ade5/execution-environment.yml#L27

The `compile` profile exists for this purpose - it will only get installed to build the wheel and will not end up in the final image. You can see an example of its usage here:

https://github.com/ansible/awx-ee/blob/71158e4db0dcfe06e5a49e123269ce76a2a5ade5/execution-environment.yml#L29